website/docs/r/elastic_cloud_elasticsearch.html.markdown

```diff
@@ -41,6 +41,10 @@ The following arguments are supported:
 
 * `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.
 
+**NOTE:** The SKU depends on the Elasticsearch Plans available for your account and is a combination of PlanID_Term.\
+Ex: If the plan ID is "planXYZ" and term is "Yearly", the SKU will be "planXYZ_Yearly".\
```

Comment:
Same here.

Reply:
It is for a new line in markdown; without it the lines look very long and clumsy.

WITHOUT '\'
![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/85922721/b30202ab-5647-4f99-a5b2-404a5a67fd97)

WITH '\'
![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/85922721/9fe52616-c969-41df-a03d-ce303fbdfede)

Reply:
I'm not sure the "NOTE" decoration will account for the new lines. So I'd suggest we keeps the oneline note as illustrated in the first picture.

Reply:
@magodo 

NOTE will not produce a new line.
Two ways that I have followed to add a new line are:
1. With \\ 
2. Without \\ but with two spaces and an enter as mentioned in https://github.com/hashicorp/terraform-provider-azurerm/pull/23863#discussion_r1389348109.

Please suggest what you prefer here.

Reply:
Without new lines the NOTE will look a tad clumsy as shown in the first picture.
If that is okay with the doc, I will remove the new lines in NOTE and make it like the first picture.
Kindly suggest.

Reply:
Yes, I believe that is how the other "NOTES" look like among the code base (you can `grep` NOTE to check). Some wordy "NOTE" example (https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/subnet):

![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/7970698/ac3e5c8e-08ad-4f2f-b3ca-c24b77366d57)

Reply:
Done
https://github.com/hashicorp/terraform-provider-azurerm/pull/23863/commits/ab8b0fa3159cadee34db9d592065b44be9accbca